Repo {repoSshUrl = "git@github.com:combro2k/fail2ban.git", repoDescription = Just "Daemon to ban hosts that cause multiple authentication errors", repoCreatedAt = GithubDate {fromGithubDate = 2012-12-04 13:28:21 UTC}, repoHtmlUrl = "https://github.com/combro2k/fail2ban", repoSvnUrl = "https://github.com/combro2k/fail2ban", repoForks = 0, repoHomepage = Just "http://www.fail2ban.org", repoFork = True, repoGitUrl = "git://github.com/combro2k/fail2ban.git", repoPrivate = False, repoCloneUrl = "https://github.com/combro2k/fail2ban.git", repoSize = 164, repoUpdatedAt = GithubDate {fromGithubDate = 2013-01-13 12:06:30 UTC}, repoWatchers = 0, repoOwner = GithubUser {githubOwnerAvatarUrl = "https://secure.gravatar.com/avatar/d170656204d6567a09ef6fb7d83e6251?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png", githubOwnerLogin = "combro2k", githubOwnerUrl = "https://api.github.com/users/combro2k", githubOwnerId = 726584, githubOwnerGravatarId = "d170656204d6567a09ef6fb7d83e6251"}, repoName = "fail2ban", repoLanguage = Just "Python", repoMasterBranch = Just "master", repoPushedAt = Just (GithubDate {fromGithubDate = 2013-01-08 14:36:35 UTC}), repoId = 7000343, repoUrl = "https://api.github.com/repos/combro2k/fail2ban", repoOpenIssues = 0, repoHasWiki = Just False, repoHasIssues = Just False, repoHasDownloads = Just True}